src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -384,6 +401,9 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
           Expand all
         </MenuItem>
         <div className="react-contextmenu-separator" />
+        <MenuItem onClick={this.handleClick} data={{ type: 'searchfox' }}>
+          Look up the function name on Searchfox
```

Comment:
FYI I tried to add https://emojipedia.org/north-east-arrow/ at the end, to represent the fact that we open in a new page; but we'd need to change the font from the one we have at the moment, and I think this is more work than what's worth. We'll see if we feel the need for it.